docs/development/testguide.rst

```diff
@@ -645,26 +645,24 @@ failure to be reported.
 Tests requiring optional dependencies
 =====================================
 
-For tests that test functions or methods that require optional
-dependencies (e.g. Scipy), pytest should be instructed to skip the
-test if the dependencies are not present. The following example shows
-how this should be done::
+For tests that test functions or methods that require optional dependencies
+(e.g. Scipy), pytest should be instructed to skip the test if the dependencies
+are not present, as the Astropy tests should succeed even if an optional
+dependency is not present. Astropy provides a list of boolean flags that test
```

Comment:
Capitalized "Astropy" is for the project, not the package.

```suggestion
are not present, as the ``astropy`` tests should succeed even if an optional
dependency is not present. ``astropy`` provides a list of boolean flags that test
```

Reply:
Ah, yes, @lglattly should shake her finger at me :) -- skipping CI for this too...but I'll wait for the last Cron test to finish.